Source/OxyPlot.Maui.Skia/PlotViewBase.cs

```diff
@@ -138,6 +141,8 @@ public void HideZoomRectangle()
     /// <param name="updateData">The update Data.</param>
     public void InvalidatePlot(bool updateData = true)
     {
+        UpdateStarted?.Invoke();
```

Comment:
This PR is essentially ok with me. Only one comment here: `UpdateStarted` should probably go into `RenderOverride` (like `UpdateFinished`), just to make sure that they are both called consistently and there are no cases where the invocation of `UpdateStarted` is missing.

For example, I see that the `Render` / `RenderOverride` method is called from two other places (without `InvalidatePlot`):
* `PlotViewBase.OnLayoutUpdated`
* `PlotView.SkElement_PaintSurface`

Reply:
Main idea was to include all stages of drawing - data update and rendering itself. If `UpdateStarted ` is called from `Render `/ `RenderOverride ` it will not include `IPlotModel.Update`. So to be consistent in that way, I think I separate these stages: `UpdateStarted ` / `UpdateFinished ` - for `IPlotModel.Update`, `RenderStarted ` / `RenderFinished ` - for `IPlotModel.Render`.

Reply:
Ok, that makes sense 👍